src/api/client/cache.ts

```diff
@@ -69,63 +76,72 @@ const createCachedAvailabilityHandler = () => ({
   },
 })
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type CachePolicyFields<T extends string> = Partial<Record<T, FieldPolicy<any> | FieldReadFunction<any>>>
```

Comment:
```suggestion
type CachePolicyFields<T extends string> = Partial<Record<T, FieldPolicy | FieldReadFunction>>
```
I think it should work, right? 
I looked in the `FieldPolicy` and `FieldReadFunction` and `any` is set as default there.